syntax = "proto3";

package fileexchange;

// Interface exported by the server.
service FileExchange {
  rpc Put(OffsetData) returns (success_failure) {}
}


message success_failure {
  int32 id = 1;
}


message OffsetData {
  repeated uint64 offsets = 1;
  repeated string values = 2;
}



